storage/pebble/lookup.go

```diff
@@ -128,3 +130,17 @@ func encodedUint64(height uint64) []byte {
 	payload := make([]byte, 0, 8)
 	return binary.BigEndian.AppendUint64(payload, height)
 }
+
+// KeyToRegisterID converts a ledger key into a register ID.
+func keyToRegisterID(key ledger.Key) (flow.RegisterID, error) {
```

Comment:
@janezpodhostnik should this function live here?

We are planning to [move all these conversion function somewhere](https://github.com/onflow/flow-go/issues/4658), right?

Reply:
Let's reuse the `LedgerKeyToRegisterID` from this PR https://github.com/onflow/flow-go/pull/4766/files